examples/save_plot/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "save_plot"
+version = "0.1.0"
+authors = ["hacknus <l_stoeckli@bluewin.ch>"]
+license = "MIT OR Apache-2.0"
+edition = "2021"
+rust-version = "1.65"
+publish = false
+
+[dependencies]
+eframe = { path = "../../crates/eframe", features = [
+    "__screenshot", # __screenshot is so we can dump a screenshot using EFRAME_SCREENSHOT_TO
+] }
+image = { version = "0.24", default-features = false, features = ["png"] }
+rfd = "0.11.0"
+tracing-subscriber = "0.3"
```

Comment:
Please use `env_logger` instead of `tracing-subscriber` (see other examples). I think this changed since you opened the PR (you may also want to re-base on latest `master` 😬)